content/zh/docs/ops/ambient/install/helm-installation/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 通过 Helm 安装
+description: 如何使用 Helm 安装 Ambient Mesh。
+weight: 4
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+本指南向您展示如何使用 Helm 安装 Ambient Mesh。
+除了 [Ambient Mesh 入门](/zh/docs/ops/ambient/getting-started/)中的演示之外，
+我们**鼓励**您依照本指南安装 Ambient Mesh。Helm 可以帮助您单独管理组件，您可以轻松地将组件升级到最新版本。
+
+## 前提条件 {#prerequisites}
+
+1. 执行任何必要的[平台特定设置](/zh/docs/setup/platform-setup/)。
+
+1. 检查 [Pod 和 Service 的要求](/zh/docs/ops/deployment/requirements/)。
+
+1. [安装 Helm 客户端](https://helm.sh/docs/intro/install/)，版本要求 3.6 或更高。
+
+1. 配置 Helm 仓库：
+
+    {{< text syntax=bash snip_id=configure_helm >}}
+    $ helm repo add istio https://istio-release.storage.googleapis.com/charts
+    $ helm repo update
+    {{< /text >}}
+
+有关命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。
```

Comment:
```suggestion
**有关 Helm 命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。**
```

Reply:
Seems the whole Chinese sentence within the ** is not allowed by some lint rule, let's try.

Reply:
Well, understand. Your choice is good.